### ISSUE TITLE
Say current version in in addition to new version

### ISSUE THREAD

**Issue by jidanni** [indywidualny/FaceSlim#258]
On the new version notice on the top of the screen, please also say the version number currently installed in addition to the version number of the new version. That way we could be sure that we really will be upgrading and not just installing the same version by accident.

**Comment by indywidualny** [comment]
Well. This information is only shown when a new version is detected (higher build number, not the same as current). So I don't really see the point of doing it. When your app is up to date you will basically not see this notice. It's no way you end up installing the same version.

**Comment by jidanni** [comment]
Yes, but 1. we are always worried. 2. there is enough room on the screen.

**Comment by indywidualny** [comment]
Ok. Maybe a good point. So I will.

**Comment by jidanni** [comment]
Maybe say
Update available:
3.2.1 -> 3.2.2
or all on one line.
